src/main/webapp/app/home/home.component.ts

```diff
@@ -91,7 +92,10 @@ export class HomeComponent implements OnInit, AfterViewChecked {
     }
 
     registerAuthenticationSuccess() {
-        this.eventManager.subscribe('authenticationSuccess', () => {
+        const subscription = this.eventManager.subscribe('authenticationSuccess', () => {
+            // We only need to authenticate once, make sure we don't run this subscription multiple times
+            this.eventManager.destroy(subscription);
+
```

Comment:
I am wondering if this could also be done with 
```suggestion
        this.eventManager.pipe(take(1)).subscribe('authenticationSuccess', () => {
```
and do the same thing more readable?

Reply:
This is not a normal observable so we can't use piping here.